app/models/pageflow/revision.rb

```diff
@@ -89,7 +89,8 @@ def find_files(model)
     end
 
     def find_file(model, id)
-      file = files(model).find(id)
+      file = files(model).find_by(id: id)
+      return unless file
       UsedFile.new(file)
     end
```

Comment:
`find_file` is also used in controllers ([example](https://github.com/codevise/pageflow/blob/6f240d5b36fb0d705f6413966804bbc450ac90e1/app/controllers/pageflow/editor/files_controller.rb#L82)) which rely on `RecordNotFound` exceptions to create 404 responses. Maybe we should add `find_file_by_id` and rename it to `find_file_by_perma_id` in the next PR?

Reply:
Good idea. Done!